src/View/SSViewer.php

```diff
@@ -165,7 +165,9 @@ public static function set_themes($themes = [])
 
     public static function add_themes($themes = [])
     {
-        SSViewer::config()->merge('themes', $themes);
+        $currentThemes = SSViewer::get_themes();
+        $finalThemes = array_merge($themes, $currentThemes);
+        SSViewer::set_themes(array_values(array_unique($finalThemes)));
```

Comment:
I'd probably add a comment about why we're using `array_values` in case it's not clear to those who come along later

Reply:
Comment added